app/controllers/admin/decisions_controller.rb

```diff
@@ -5,21 +5,22 @@ def new
   end
 
   def create
-    @split = Split.find params[:split_id]
-    Split.transaction do # FIXME: This makes sure this controller either succeeds or fails atomically until we can async the work
-      @decision = @split.create_decision!(create_params)
-    end
-    flash[:success] = "Reassigned #{@decision.count} visitors to #{@decision.variant}"
-    redirect_to admin_split_path(@split)
+    CreateDecisionJob.perform_later(split, variant: target_variant, admin: current_admin)
+    flash[:success] = "Queued decision to reassign #{affected_assignments_count} visitors to #{target_variant}"
+    redirect_to admin_split_path(split)
   end
```

Comment:
@jmileham rubocop wouldn't leave me alone about the ABC size of this method until i moved all but one of the assignments into a method... so i did that. we lose the "grab the count before we delay the work" thing though. should we just disable the cop on this method?